### ISSUE TITLE
PHP Notice from nightly.php --hide-zeros

### ISSUE THREAD

**Issue by kenirwin** [suma-project/Suma#74]
Running `php nightly.php --hide-zeros`, I get this notice where there's no data to show:
```
Head Counts
No data to show for this initiative
PHP Notice:  Undefined offset: 1 in /var/www/app/suma/analysis/src/lib/php/NightlyData.php on line 344
```
I'll see if I can weed out this notice while I'm working on other report updates.

**Comment by kenirwin** [comment]
I should mention, since I'm adding a bunch of content to NightlyData.php and the line numbers will move around: the offending line is ` if (is_null($columnCounts[$key]))` in ` public function hideZeroColumns($table)`

**Comment by kenirwin** [comment]
This is fixed in pull request #75
